sdk-python/examples/error_handling/example_error_handling.py

```diff
@@ -0,0 +1,56 @@
+import asyncio
+import logging
+from pathlib import Path
+
+import littlehorse
+from littlehorse.config import LHConfig
+from littlehorse.worker import LHTaskWorker
+from littlehorse.workflow import LHErrorType
+from littlehorse.workflow import ThreadBuilder, Workflow
+
+logging.basicConfig(level=logging.INFO)
+
+
+def get_config() -> LHConfig:
+    config = LHConfig()
+    config_path = Path.home().joinpath(".config", "littlehorse.config")
+    if config_path.exists():
+        config.load(config_path)
+    return config
+
+
+def entrypoint(thread: ThreadBuilder) -> None:
+    def exception_handler(thread: ThreadBuilder) -> None:
+        thread.execute("handler")
+
+    node = thread.execute("fail")
+    thread.handle_error(node, exception_handler, LHErrorType.TASK_ERROR)
```

Comment:
Nit pick: in the other SDK's, the order of the arguments is:
1. `NodeOutput`
2. Error type
3. Error handler function

Reply:
let me check

Reply:
that is true but now error type defaults None, so it has to be at the end